test/Confluent.Kafka.StrongName.UnitTests/Confluent.Kafka.StrongName.UnitTests.csproj

```diff
@@ -24,7 +24,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="librdkafka.redist" Version="0.11.1" />
+    <PackageReference Include="librdkafka.redist" Version="0.11.1-TEST101" />
```

Comment:
dito, when change

Reply:
we can't change this until librdkafka.redist 0.11.3 is live on nuget. any time after that.